### PR TITLE
CRIMRE-273 - Changed wording from 'View all open applications' link to 'Check all open applications' link on the 'Your list' page

### PR DESCRIPTION
**Pull request by MohammedHassanLatif93**
## Description of change
Changed wording from 'View all open applications' link to 'Check all open applications' link on the 'Your list' page

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-273

### Before changes:
image

### After changes:
image

## How to manually test the feature
- User logs in the portal
- User will see the 'Your list' page
- User will see the 'Check all open applications' link as shown above in the After changes screenshot

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -206,7 +206,7 @@ en:
     reassign_to_self: Reassign to myself
     search: Search
     unassign_from_self: Remove from your list
-    view_all_open_applications: View all open applications
+    view_all_open_applications: Check all open applications
```
Comment (arthurashman) [inline config/locales/en.yml:209]: This is actually not used at all and can be completely deleted

File: spec/system/assigning/viewing_your_assigned_applications_spec.rb [hunk 2]
```diff
@@ -89,13 +89,13 @@
...
-  context 'when using the all applications link' do
+  context 'when using the check all applications link' do
     before do
-      click_on('View all open applications')
+      click_on('Check all open applications')
     end
 
     it 'proceeds to the correct page' do
-      expect(page).to have_content 'All open applications'
+      expect(page).to have_content 'Check all open applications'
```
Comment (arthurashman) [inline spec/system/assigning/viewing_your_assigned_applications_spec.rb:98]: So this is not the link that you've changed. This is the title of the page that the link goes to. 

If you read the test, it goes:
Test to check use of check all applications link
Click on 'check all applications'
It goes to the right page
Check it goes to the right page by checking the title of the page we got redirected to

File: spec/system/sub_nav_spec.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
   before do
     visit '/'
-    click_on 'All open applications'
+    click_on 'Check all open applications'
```
[the remark this answers is not in the document]
Comment (MohammedHassanLatif93) [reply spec/system/sub_nav_spec.rb:8]: No problem at all! I've changed this back to All open applications 👍

**Review by arthurashman — APPROVED** [review]
Ace, great job 🎉

**Pull request merged**